e2e/cypress/tests/integration/system_console/ui_and_api/custom_site_name_description_spec.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @customization
+
+describe('Customization', () => {
+    it('MM-Txxx - Should match title and custom description in root html', () => {
```

Comment:
Will update test key once https://github.com/mattermost/mattermost-test-management/pull/58 is merged and saved to Zephyr.